pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -117,27 +117,43 @@ type ServiceClass struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// BrokerName is the reference to the Broker that provides this service.
+	// BrokerName is the reference to the Broker that provides this
+	// ServiceClass.
 	// Immutable.
 	BrokerName string `json:"brokerName"`
 
-	// Bindable indicates whether a user can create bindings to an instance of
-	// this service. ServicePlan has an optional field called Bindable which
-	// overrides the value of this field.
-	Bindable      bool          `json:"bindable"`
-	Plans         []ServicePlan `json:"plans"`
-	PlanUpdatable bool          `json:"planUpdatable"` // Do we support this?
+	// Description is a short description of this ServiceClass.
+	Description string `json:"description"`
+
+	// Bindable indicates whether a user can create bindings to an Instance
+	// provisioned from this service. ServicePlan has an optional field called
+	// Bindable which overrides the value of this field.
+	Bindable bool `json:"bindable"`
+
+	// Plans is the list of ServicePlans for this ServiceClass.  All
+	// ServiceClasses have at least one ServicePlan.
```

Comment:
We do not, and we should, because it is not optional in the API.  And, if we accept a serviceclass without any plans, you won't be able to provision it, anyway.  Are you cool with an immediate follow-up for that?